src/components/PlioQuestion.vue

```diff
@@ -99,13 +108,17 @@ import { faCheckCircle } from '@fortawesome/free-solid-svg-icons/faCheckCircle';
 import { faTimesCircle } from '@fortawesome/free-solid-svg-icons/faTimesCircle';
 library.add(faWindowClose, faCheckCircle, faTimesCircle);
 
+import SubmitButtonPointer from './tutorial/SubmitButtonPointer.vue';
+import mcqOptionsPointer from './tutorial/mcqOptionsPointer.vue';
+import CloseButtonPointer from './tutorial/CloseButtonPointer.vue';
```

Comment:
Importing these tutorial components

Reply:
Each pointer is separate component? Can you make a pointer component, and keep the icon, and position as parameter?